### PR TITLE
doc: update Digital Twin KIT

### PR DESCRIPTION
**Pull request by BirgitBoss**
Minor updates to Digital Twin KIT
- update some links
- editorial changes

### CODE REVIEW

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 1]
```diff
@@ -443,14 +449,14 @@ Data Providers will usually follow one of two patterns:
...
-   requests to the respective backend systems. This is particularly feasible in the Catena-X dataspace since it
+   requests to the respective backend systems. This is particularly feasible in the Catena-X dataspace.
 
 Offering data to the network requires mappings that are naturally dependent on the data source format. More on data integration
 can be found in the corresponding [CX e.V. guide](https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Onboarding/DataIntegrationPatterns_Guide_Final_V1.pdf).
 
 ### Patterns for DTRs
 
-Usually, a DTR will implement a persistence with the specified AAS-APIs for data ingestion specified in the SSP-001 by
+Usually, a DTR will implement a persistence with the specified AAS-APIs for data ingestion specified in the SSP-0001 by
```
Comment (arnoweiss) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:459]: I'm pretty sure the profiles only have three-digit ranges.

**Review by arnoweiss — APPROVED** [review]
looks good - thanks for the bugfixes.

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md (hunk 1 above)
Comment (BirgitBoss) [reply docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:459]: This is true

**Review by arnoweiss — APPROVED** [review]

File: docs-kits/kits/Digital Twin Kit/page_software-operation-view.md [hunk 2]
```diff
@@ -18,13 +18,15 @@ Digital Twins the Catena-X-way. This infrastructure empowers Data Consumers to c
...
+| Digital Twin Registry | A registry for digital twins, each digital twin providing the endpoints of its submodels. <br> The registry is adhering to the AAS Registry API. | [Tractus-X Digital Twin Registry](https://github.com/eclipse-tractusx/sldt-digital-twin-registry)                                                                                                   | CX - 0002                                                   |
+| Submodel Server       | The data source adhering to a subset of the Submodel API as defined in AAS Part-2 3.0.                   | [FA³ST-Framework](https://github.com/FraunhoferIOSB/FAAAST-Service), <br> [Eclipse Basyx](https://github.com/eclipse-basyx/basyx-java-sdk), <br> [AASX Server](https://github.com/admin-shell-io/aasx-server) | CX - 0002                                                   |
```
Comment (arnoweiss) [inline docs-kits/kits/Digital Twin Kit/page_software-operation-view.md:29]: Line  28 and 29 make the build fail. For a line break in a table, the self-closing tag `<br/>` does that. The lone opening tag `<br>` doesn't work.

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 3]
```diff
@@ -59,25 +59,30 @@ their DTR in any way they desire.
...
+same capability as a central [Digital Twin Registry](#digital-twin-registry): 
+It allows to start a Discovery Process with an Asset ID. 
 However, in Catena-X some of the data is deemed so sensitive that a central authority can't be
-trusted with it. Thus, each Data Provider will run their own DTR which poses a challenge for discovery. After all, a
+trusted with it. Thus, a decentralized approach is implemented: each Data Provider will run their own DTR. 
+This poses a challenge for discovery if the BPN of the supplier is not known by the data consumer. After all, a
 Data Consumer must still find out the address where to fetch the data from. That's why Catena-X has introduced a 
 three-step discovery pattern made up of the central microservices Discovery Finder, BPN Discovery (or several of them) 
-and finally the EDC discovery that is part of the CX-Portal. They are also part of this Kit.
+and finally the EDC discovery that is part of the CX-Portal.
```
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:75]:
```suggestion
and finally the EDC discovery that is part of the CX-Portal.
```

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 4]
```diff
@@ -59,25 +59,30 @@ their DTR in any way they desire.
...
+It allows to start a Discovery Process with an Asset ID. 
 However, in Catena-X some of the data is deemed so sensitive that a central authority can't be
-trusted with it. Thus, each Data Provider will run their own DTR which poses a challenge for discovery. After all, a
+trusted with it. Thus, a decentralized approach is implemented: each Data Provider will run their own DTR. 
+This poses a challenge for discovery if the BPN of the supplier is not known by the data consumer. After all, a
 Data Consumer must still find out the address where to fetch the data from. That's why Catena-X has introduced a 
 three-step discovery pattern made up of the central microservices Discovery Finder, BPN Discovery (or several of them) 
-and finally the EDC discovery that is part of the CX-Portal. They are also part of this Kit.
+and finally the EDC discovery that is part of the CX-Portal. 
+The Discovery Finder and the BPN Discovery service are described as part of this Kit.
```
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:76]: Again a space at the end of the line ;-)
If we already refer to the documentation of these services, shouldn't we directly provide a link?
```suggestion
The Discovery Finder and the BPN Discovery service are described as part of this Kit.
```

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 5]
```diff
@@ -59,25 +59,30 @@ their DTR in any way they desire.
...
+and finally the EDC discovery that is part of the CX-Portal. 
+The Discovery Finder and the BPN Discovery service are described as part of this Kit. 
 
 ## Discovery Sequence
 
 The services that make up the Digital Twin Kit partly rely on each other. Executing them in the right sequence allows 
 a Data Consumer to get access to data whose location was previously unknown. However, this discovery process relies on a 
-set of assumptions. Most relevant is the presence of the discovery services defined in CX - 0053 ([Discovery Finder](./API%20Discovery%20Finder/discovery-finder.info.mdx), 
-[BPN Discovery](./API%20BPN%20Discovery/bpn-discovery-service.info.mdx)) and CX - 0001 ([EDC Discovery](./API%20EDC%20Discovery/post-list-of-bpns-or-an-empty-array-to-retrieve-available-company-connector-authorization-required-roles-view-connectors.api.mdx)). 
+set of assumptions. Most relevant is the presence of the discovery services defined in
```
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:82]:
```suggestion
set of assumptions. Most relevant is the presence of the discovery services defined in
```

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 6]
```diff
@@ -59,25 +59,30 @@ their DTR in any way they desire.
...
+The Discovery Finder and the BPN Discovery service are described as part of this Kit. 
 
 ## Discovery Sequence
 
 The services that make up the Digital Twin Kit partly rely on each other. Executing them in the right sequence allows 
 a Data Consumer to get access to data whose location was previously unknown. However, this discovery process relies on a 
-set of assumptions. Most relevant is the presence of the discovery services defined in CX - 0053 ([Discovery Finder](./API%20Discovery%20Finder/discovery-finder.info.mdx), 
-[BPN Discovery](./API%20BPN%20Discovery/bpn-discovery-service.info.mdx)) and CX - 0001 ([EDC Discovery](./API%20EDC%20Discovery/post-list-of-bpns-or-an-empty-array-to-retrieve-available-company-connector-authorization-required-roles-view-connectors.api.mdx)). 
+set of assumptions. Most relevant is the presence of the discovery services defined in 
+CX - 0053 ([Discovery Finder](./API%20Discovery%20Finder/discovery-finder.info.mdx),
```
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:83]:
```suggestion
CX - 0053 ([Discovery Finder](./API%20Discovery%20Finder/discovery-finder.info.mdx),
```

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 7]
```diff
@@ -59,25 +59,30 @@ their DTR in any way they desire.
...
 
 ## Discovery Sequence
 
 The services that make up the Digital Twin Kit partly rely on each other. Executing them in the right sequence allows 
 a Data Consumer to get access to data whose location was previously unknown. However, this discovery process relies on a 
-set of assumptions. Most relevant is the presence of the discovery services defined in CX - 0053 ([Discovery Finder](./API%20Discovery%20Finder/discovery-finder.info.mdx), 
-[BPN Discovery](./API%20BPN%20Discovery/bpn-discovery-service.info.mdx)) and CX - 0001 ([EDC Discovery](./API%20EDC%20Discovery/post-list-of-bpns-or-an-empty-array-to-retrieve-available-company-connector-authorization-required-roles-view-connectors.api.mdx)). 
+set of assumptions. Most relevant is the presence of the discovery services defined in 
+CX - 0053 ([Discovery Finder](./API%20Discovery%20Finder/discovery-finder.info.mdx), 
+[BPN Discovery](./API%20BPN%20Discovery/bpn-discovery-service.info.mdx))
```
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:84]:
```suggestion
[BPN Discovery](./API%20BPN%20Discovery/bpn-discovery-service.info.mdx))
```

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 8]
```diff
@@ -59,25 +59,30 @@ their DTR in any way they desire.
...
 ## Discovery Sequence
 
 The services that make up the Digital Twin Kit partly rely on each other. Executing them in the right sequence allows 
 a Data Consumer to get access to data whose location was previously unknown. However, this discovery process relies on a 
-set of assumptions. Most relevant is the presence of the discovery services defined in CX - 0053 ([Discovery Finder](./API%20Discovery%20Finder/discovery-finder.info.mdx), 
-[BPN Discovery](./API%20BPN%20Discovery/bpn-discovery-service.info.mdx)) and CX - 0001 ([EDC Discovery](./API%20EDC%20Discovery/post-list-of-bpns-or-an-empty-array-to-retrieve-available-company-connector-authorization-required-roles-view-connectors.api.mdx)). 
+set of assumptions. Most relevant is the presence of the discovery services defined in 
+CX - 0053 ([Discovery Finder](./API%20Discovery%20Finder/discovery-finder.info.mdx), 
+[BPN Discovery](./API%20BPN%20Discovery/bpn-discovery-service.info.mdx)) 
+and CX - 0001 ([EDC Discovery](./API%20EDC%20Discovery/post-list-of-bpns-or-an-empty-array-to-retrieve-available-company-connector-authorization-required-roles-view-connectors.api.mdx)).
```
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:85]:
```suggestion
and CX - 0001 ([EDC Discovery](./API%20EDC%20Discovery/post-list-of-bpns-or-an-empty-array-to-retrieve-available-company-connector-authorization-required-roles-view-connectors.api.mdx)).
```

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 9]
```diff
@@ -419,18 +425,18 @@ these are the layers of complexity:
...
-| Data Model Version | 5, 21, 29                                                   | The structure of the Submodel is determined by the aspect-model's URN. It includes a section on versioning.                                                                          | A new version of the schema requires setup of a new submodel. This includes registration at the DTR. |
+| Dataspace Protocol | 12, 22                                                      | The body of the `{{consumerControlPlane}}/v2/catalog/request`-request points to a versioned endpoint of a business partner's DSP endpoint like `{{providerControlPlane}}/api/v1/dsp` | under discussion in Connector Kit.                                                          |
+| AAS Specification  | 4, 5, 15, 25                                                | For all AAS-related EDC-Assets (styled as dcat:Dataset in the catalog), a property cx-common:version must be added referring to the major and minor version of the AAS-spec.         |  under discussion in DT Kit.                                                                 |
```
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:429]: To keep it consistent I would start the fourth column also with an uppercase.

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 10]
```diff
@@ -443,14 +449,15 @@ Data Providers will usually follow one of two patterns:
...
-   requests to the respective backend systems. This is particularly feasible in the Catena-X dataspace since it
+   requests to the respective backend systems. This is particularly feasible in the Catena-X dataspace.
 
 Offering data to the network requires mappings that are naturally dependent on the data source format. More on data integration
 can be found in the corresponding [CX e.V. guide](https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Onboarding/DataIntegrationPatterns_Guide_Final_V1.pdf).
 
 ### Patterns for DTRs
 
-Usually, a DTR will implement a persistence with the specified AAS-APIs for data ingestion specified in the SSP-001 by
+Usually, a DTR will implement a persistence with the specified AAS-APIs for data ingestion specified in the
```
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:459]:
```suggestion
Usually, a DTR will implement a persistence with the specified AAS-APIs for data ingestion specified in the
```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]
Many thanks for the changes! To me, the changes seem to be very beneficial for the users.

I would recommend you to configure your IDE/editor to automatically remove the spaces at the end ;-) Behind this link you can find instructions on how to do that for vscode: https://stackoverflow.com/a/30884298

File: docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md [hunk 11]
```diff
@@ -59,25 +59,30 @@ their DTR in any way they desire.
 
 DTRs hold sensitive information: a SubmodelDescriptor may not give access to the actual Submodel-data but all in cumulo hint at 
 production volumes as each Twin represents an asset. Therefore, Catena-X implements decentral DTRs (DDTR), each running with a 
-business partner. [In an IDTA-Whitepaper](https://industrialdigitaltwin.org/en/wp-content/uploads/sites/2/2023/06/Decentralized-Registries-Taxonomy-of-decentralized-registries-and-an-architectural-overview_.pdf ),
+business partner.
+[In an IDTA-Whitepaper](https://industrialdigitaltwin.org/en/wp-content/uploads/sites/2/2023/06/Decentralized-Registries-Taxonomy-of-decentralized-registries-and-an-architectural-overview_.pdf ),
```
[the remark this answers is not in the document]
Comment (BirgitBoss) [reply docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md:63]: Thanks for the hints with the spaces

**Comment by BirgitBoss** [comment]
@florianrusch-zf Thanks for your careful review. I followed all of your suggestions. Now it should be fine.

File: docs-kits/kits/Digital Twin Kit/page_software-operation-view.md [hunk 12]
```diff
@@ -18,13 +18,15 @@ Digital Twins the Catena-X-way. This infrastructure empowers Data Consumers to c
...
+| Digital Twin Registry | A registry for digital twins, each digital twin providing the endpoints of its submodels. <br> The registry is adhering to the AAS Registry API. | [Tractus-X Digital Twin Registry](https://github.com/eclipse-tractusx/sldt-digital-twin-registry)                                                                                                   | CX - 0002                                                   |
+| Submodel Server       | The data source adhering to a subset of the Submodel API as defined in AAS Part-2 3.0.                   | [FA³ST-Framework](https://github.com/FraunhoferIOSB/FAAAST-Service), <br/> [Eclipse Basyx](https://github.com/eclipse-basyx/basyx-java-sdk), <br> [AASX Server](https://github.com/admin-shell-io/aasx-server) | CX - 0002                                                   |
```
Comment (florianrusch-zf) [inline docs-kits/kits/Digital Twin Kit/page_software-operation-view.md:29]: As @arnoweiss pointed out, line breakes in a tables need to be `<br/>` and NOT `<br>`.

```suggestion
| Digital Twin Registry | A registry for digital twins, each digital twin providing the endpoints of its submodels. <br/> The registry is adhering to the AAS Registry API. | [Tractus-X Digital Twin Registry](https://github.com/eclipse-tractusx/sldt-digital-twin-registry)                                                                                                   | CX - 0002                                                   |
| Submodel Server       | The data source adhering to a subset of the Submodel API as defined in AAS Part-2 3.0.                   | [FA³ST-Framework](https://github.com/FraunhoferIOSB/FAAAST-Service), <br/> [Eclipse Basyx](https://github.com/eclipse-basyx/basyx-java-sdk), <br/> [AASX Server](https://github.com/admin-shell-io/aasx-server) | CX - 0002                                                   |
```

**Comment by danielmiehle** [comment]
@BirgitBoss @arnoweiss : is this PR still valid after the release? or can we close it?

**Comment by arnoweiss** [comment]
All relevant changes were accepted by https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/654

**Pull request closed without merging**